components/lib/crash/src/main/java/mozilla/components/lib/crash/service/SentryService.kt

```diff
@@ -90,13 +90,18 @@ class SentryService(
 
     override fun report(crash: Crash.NativeCodeCrash): String? {
         if (sendEventForNativeCrashes) {
+            val level = when (crash.isFatal) {
+                true -> Event.Level.FATAL
+                else -> Event.Level.ERROR
+            }
+
             crash.breadcrumbs.forEach {
                 client.context.recordBreadcrumb(it.toSentryBreadcrumb())
             }
 
             val eventBuilder = EventBuilder()
                 .withMessage(createMessage(crash))
-                .withLevel(Event.Level.INFO)
+                .withLevel(level)
```

Comment:
The initial idea here was that this is just a "log message" as opposed to an actual crash report that we can act on. But I guess we can also send it with "error" level if you think that is more helpful.

Reply:
I agree.  But there are some issues where other teams thought native crashes are not actual crashes.  Because caught exceptions are also sent as INFO.  This will make it easier to spot real crashes in Sentry. 👍